tests/conftest.py

```diff
@@ -197,3 +201,8 @@ def cleanup():
     email = "lektortest@example.com"
     subprocess.check_call(['git', 'config', 'user.email', email])
     return email
+
+
+@pytest.fixture(scope='session')
+def splinter_screenshot_dir(request):
```

Comment:
I suggest you use the [tmpdir](http://doc.pytest.org/en/latest/tmpdir.html) fixture, which is built-in to pytest.